content/for-developers/installation.mdx

```diff
@@ -61,7 +61,7 @@ If you are adding Decanter to an existing project please have a review of the [F
 <div className="children:children:rs-mb-4">
 1. **Install Decanter Tailwindcss**
 
-    The easiest way to add Decanter to your project is to use the npm package manager.  
+    The easiest way to add Decanter to your project is to use the npm package manager.
```

Comment:
It's 2024 and I'm submitting PRs removing trailing whitespace.

Reply:
https://github.com/mattcone/markdown-guide/blob/master/_basic-syntax/line-breaks.md#line-break-best-practices

> You can use two or more spaces (commonly referred to as "trailing whitespace") for line breaks in nearly every Markdown application, but it's controversial. It's hard to see trailing whitespace in an editor, and many people accidentally or intentionally put two spaces after every sentence. For this reason, you may want to use something other than trailing whitespace for line breaks. If your Markdown application [supports HTML](https://github.com/mattcone/markdown-guide/blob/master/basic-syntax/#html), you can use the <br> HTML tag.

So you should be replacing the two spaces with a`<br />` in this case.

Reply:
TIL